ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
If we raise an exception in `update_config`, it won't work when doing `paster less`, because the `ckan.base_public_folder` config is used there.